docs/install.md

```diff
@@ -0,0 +1,8 @@
+Installing LibreTime
+====================
+
+LibreTime should generally be installed on a dedicated host. By default, its installer will install and configure all its dependencies.
```

Comment:
Doesn't this need at least a small warning that it currently only runs on Ubuntu trusty and that it might trash your system? The interactive method is not very interactive and IMO sudos too much before giving me a change to review what is being done.